CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Mixxx Contribution Guidelines #
+Thank you for contributing to [Mixxx](https://mixxx.org/)! Your work helps DJs all over the world! We are global, all volunteer team that works by consensus and we are excited to have you join us. This document specifies technical aspects of our workflow. For social aspects please refer to [CODE_OF_CONDUCT.md](https://github.com/mixxxdj/mixxx/blob/master/CODE_OF_CONDUCT.md) in this repository. We encourage you to introduce yourself on our [Zulip chat](https://mixxx.zulipchat.com/) before starting to contribute code to Mixxx.
+
+Table of Contents
+1. [Orientation](#Orientation)
+   1. [Git Repositories](#Git-Repositories)
+2. [Git Workflow](#Git-Workflow)
+   1. [All Contributors](#All-Contributors)
+   2. [Core Team](#Core-Team)
```

Comment:
The numbers here don't make much sense to me

Reply:
Are you suggesting to change this to an unordered list? Tables of content are usually ordered lists.

Reply:
On github mostly not, from what I've seen, for example:
https://github.com/sghpjuikit/player/blob/master/README.md
https://github.com/sharkdp/bat/blob/master/README.md

So, yes, I would change this to an unordered list, and maybe even remove the heading "Table of Contents" - it is self-evident.